src/ContentFormatter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lcobucci\ContentNegotiation;
+
+use Psr\Http\Message\StreamFactoryInterface;
+use Psr\Http\Message\StreamInterface;
+
+abstract class ContentFormatter extends BodyOnlyFormatter
```

Comment:
I don't see the need for two abstract classes for this...

Reply:
I've tried my best :) It's up to you how you see it.

I would keep `ContentFormatter` if you want to have only one.

Reply:
It's alright. That's what I'm doing locally, don't worry :+1:

Reply:
Great :+1: